src/components/network-chart/index.css

```diff
@@ -54,23 +54,85 @@
 }
 
 .networkChartRowItemBar {
-  background-color: var(--blue-50);
+  background-color: white;
+  border: 1px solid var(--grey-40);
   display: inline-block;
   z-index: 3;
   border-radius: 2px;
   height: 12px;
   flex-grow: 1;
   position: absolute;
   margin-top: 2px;
+  opacity: 0.7;
+}
+
+.networkChartRowItemBarInner {
+  height: 12px;
+  display: inline-block;
+  background-color: var(--grey-40);
+}
+
+.css .networkChartRowItemBar {
+  border-color: var(--purple-50);
+  opacity: 0.6;
+}
+
+.css .networkChartRowItemBarInner {
+  background-color: var(--purple-50);
+}
+
+.js .networkChartRowItemBar {
+  border-color: var(--yellow-50);
+  opacity: 0.6;
+}
+
+.js .networkChartRowItemBarInner {
+  background-color: var(--yellow-50);
+}
+
+.html .networkChartRowItemBar {
+  border-color: var(--blue-60);
   opacity: 0.6;
 }
 
-.STATUS_START .networkChartRowItemBar {
-  background-color: var(--magenta-50);
-  opacity: 0.2;
+  .html .networkChartRowItemBarInner {
+    background-color: var(--blue-60);
+  }
+
+.img .networkChartRowItemBar {
+  border-color: var(--green-60);
+  opacity: 0.6;
+}
+
+  .img .networkChartRowItemBarInner {
+    background-color: var(--green-60);
+  }
+
+.request-queue,
```

Comment:
Please prefix these and camel case to follow the existing class naming system, like `networkChartRowItemRequestQueue`.